src/hooks/useSubaccount.tsx

```diff
@@ -291,24 +294,27 @@ const useSubaccountContext = ({ localDydxWallet }: { localDydxWallet?: LocalWall
           amount: balanceAmount - AMOUNT_RESERVED_FOR_GAS_USDC,
           subaccountClient,
         });
+        dispatch(depositsSweptIntoSubaccount({ dydxAddress }));
       } else if (shouldWithdraw) {
         await withdrawFromSubaccount({
           amount: AMOUNT_RESERVED_FOR_GAS_USDC - balanceAmount,
           subaccountClient,
         });
       }
     },
-    [subaccountClient, depositToSubaccount, withdrawFromSubaccount]
+    [subaccountClient, depositToSubaccount, dispatch, dydxAddress, withdrawFromSubaccount]
   );
 
   const balances = useAppSelector(BonsaiCore.account.balances.data);
   const usdcCoinBalance = balances.usdcAmount;
 
+  const pendingDeposits = useParameterizedSelector(selectPendingDeposits, dydxAddress);
+
   useEffect(() => {
-    if (usdcCoinBalance && !isKeplr) {
+    if (usdcCoinBalance && !isKeplr && !pendingDeposits.length) {
```

Comment:
grrr @tyleroooo this introduces a case where if you do a slow deposit and then an instant deposit, the instant deposit won't be swept and considered "done" until the slow deposit is confirmed by skip. need to noodle on this more